src/main/java/com/_2rkdev/dischard/dto/common/Member.java

```diff
@@ -0,0 +1,9 @@
+package com._2rkdev.dischard.dto.common;
+
+public record Member(
```

Comment:
prefer to add the suffix DTO for clarity. 
This should be:
```suggestion
public record MemberDTO(
```